sap/adt/core.py

```diff
@@ -285,11 +285,15 @@ def execute(self, method, adt_uri, params=None, headers=None, body=None, accept=
 
         return resp
 
-    def get_text(self, relativeuri):
-        """Executes a GET HTTP request with the headers Accept = text/plain.
+    def get_text(self, relativeuri, headers=None):
+        """Executes a GET HTTP request with the headers Accept = ....
+            If headers are not provided the text/plain is set by default
         """
 
-        return self.execute('GET', relativeuri, headers={'Accept': 'text/plain'}).text
+        if headers is None:
+            headers = {'Accept': 'text/plain'}
+
+        return self.execute('GET', relativeuri, headers=headers).text
```

Comment:
Please add this to a solo commit with the changes in sap/adt/object.py and carefully explain the reason for the change in commit mesage.

Reply:
Removed in new correction since it is not needed any more.